docs/Makefile

```diff
@@ -53,6 +53,7 @@ clean:
 .PHONY: html
 html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	mkdir -p $(BUILDDIR)/html/docs && ln -sfn ../_static $(BUILDDIR)/html/docs/_static
```

Comment:
this is needed because `README.md` points to `docs/_static/img` and therefore, `index.rst` needs to point to `docs/_static/img` correctly as well.